plugins/1.auth0.client.ts

```diff
@@ -9,7 +9,7 @@ export default defineNuxtPlugin((nuxtApp) => {
     cacheLocation: "localstorage",
     authorizationParams: {
       redirect_uri: window.location.origin,
-      audience: runtimeConfig.public.apiUrl,
+      audience: "https://bmm-api.brunstad.org",
```

Comment:
What is the reason for this? Isn't it ok to keep it in the runtime config? I guess the url for the API doesn't change

Reply:
auth0 should always use bmm-api even when the apiUrl changes to int-bmm-api.brunstad.org.

I used int-bmm-api for this PR since on INT we show an example of all possible recommendations.